src/System.Net.Primitives/src/System/Net/CredentialCache.cs

```diff
@@ -144,13 +145,6 @@ public void Remove(string host, int port, string authenticationType)
             _cacheForHosts.Remove(key);
         }
 
-        /// <devdoc>
```

Comment:
Same as above and throughout the document: I remember we're required to leave these in place.